perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py

```diff
@@ -0,0 +1,303 @@
+                "split_points_distance_tolerance": 0.2,
+                "split_height_distance": 0.2,
+                "use_virtual_ground_point": False,
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
+    load_laserscan_to_occupancy_grid_map = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                FindPackageShare("laserscan_to_occupancy_grid_map"),
+                "/launch/laserscan_to_occupancy_grid_map.launch.py",
+            ]
+        ),
+        launch_arguments={
+            "container": "/perception/object_segmentation/ground_segmentation/perception_pipeline_container",
+            "input/obstacle_pointcloud": "no_ground/oneshot/pointcloud",
+            "input/raw_pointcloud": "/sensing/lidar/pointcloud_preprocessor/concatenated/pointcloud",
+            "output": "occupancy_grid",
+            "use_intra_process": LaunchConfiguration("use_intra_process"),
+        }.items(),
+    )
+
+    occupancy_grid_map_outlier_component = ComposableNode(
+        package="pointcloud_preprocessor",
+        plugin="pointcloud_preprocessor::OccupancyGridMapOutlierFilterComponent",
+        name="occupancy_grid_map_outlier_filter",
+        remappings=[
+            ("~/input/occupancy_grid_map", "occupancy_grid"),
+            ("~/input/pointcloud", "no_ground/oneshot/pointcloud"),
+            ("~/output/pointcloud", "/perception/object_segmentation/pointcloud"),
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
+    # set container to run all required components in the same process
+    container = ComposableNodeContainer(
+        name="perception_pipeline_container",
+        namespace="",
+        package="rclcpp_components",
+        executable=LaunchConfiguration("container_executable"),
+        composable_node_descriptions=[
+            cropbox_component,
+            occupancy_grid_map_outlier_component,
+        ],
+        output="screen",
+    )
+
+    additional_pipeline_loader = LoadComposableNodes(
+        composable_node_descriptions=additional_pipeline_components,
+        target_container=container,
+        condition=IfCondition(
+            LaunchConfiguration(
+                "use_additional_pipeline", default=pipeline_param["use_additional_pipeline"]
+            )
+        ),
+    )
+
+    customized_component_loader = LoadComposableNodes(
+        composable_node_descriptions=[customized_ground_component, ground_concat_component],
+        target_container=container,
+        condition=IfCondition(
+            LaunchConfiguration(
+                "use_additional_pipeline", default=pipeline_param["use_additional_pipeline"]
+            )
+        ),
+    )
+
+    reference_component_loader = LoadComposableNodes(
+        composable_node_descriptions=[ground_component],
```

Comment:
What is `ground_component`?

Reply:
It is a ground filter node, but the name is not readable.
fix components name in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/117/commits/3f2a0dbd8c0bcb84a7121da15438c816b4b67970

Reply:
fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/117/commits/732f49b305177898d8256dd8699b871997f04418